islandora.module

```diff
@@ -58,7 +58,9 @@ define('ISLANDORA_DATASTREAM_INGESTED_HOOK', 'islandora_datastream_ingested');
 define('ISLANDORA_DATASTREAM_MODIFIED_HOOK', 'islandora_datastream_modified');
 define('ISLANDORA_DATASTREAM_PURGED_HOOK', 'islandora_datastream_purged');
 define('ISLANDORA_INGEST_STEP_HOOK', 'islandora_ingest_steps');
+// ISLANDORA_DERVIATIVE_CREATION_HOOK is depricated. Should be removed after next release.
 define('ISLANDORA_DERVIATIVE_CREATION_HOOK', 'islandora_derivative');
```

Comment:
To avoid any potential issue for different values being introduced, it might be desirable to instead put:
```php
define('ISLANDORA_DERVIATIVE_CREATION_HOOK', ISLANDORA_DERIVATIVE_CREATION_HOOK);
```
below the definition of the correct constant (would also likely ways to move the comment, of course).